### ISSUE TITLE
[homepage refresh] clean up "no longer used" code for the homepage

### ISSUE THREAD

**Issue by Pomax** [mozilla/foundation.mozilla.org#5037]
- [ ] clean up the homepage template to make sure no old code persists
- [ ] audit which page models are no longer needed
   - [ ] clean those up

**Comment by Pomax** [comment]
blocked on:
- https://github.com/mozilla/foundation.mozilla.org/pull/5015
- https://github.com/mozilla/foundation.mozilla.org/pull/5013

**Comment by cadecairos** [comment]
merged
